lib/Twig/Extension/Profiler.php

```diff
@@ -11,11 +11,11 @@
 
 class Twig_Extension_Profiler extends Twig_Extension
 {
-    private $actives;
+    private $actives = [];
```

Comment:
given we initialize it with a non-empty array, I don't think this is worth it. It means doing an extra array mutation (you create the empty array first and then you alter it)